setup.sh

```diff
@@ -47,9 +60,9 @@ function clean_legacy()
   local completely_remove=$1
 
   local toDelete="$APPLICATIONNAME"
```

Comment:
Hmm, this is all changed in the next commit, so I don't feel like we need to rename it now. But I can do it, if you think it would be better for the commit history.

Reply:
I did not noticed that you changed it in the next commit, just ignore my comment.